lib/engine/config/game/g_1836_jr30.rb

```diff
@@ -286,9 +286,14 @@ module G1836Jr30
             {
                "type":"tile_lay",
                "owner_type":"corporation",
+               "free_tile":"true",
```

Comment:
we can just call this "free": true

no quotes around true

Reply:
okay, I fixed that